### ISSUE TITLE
Nested modals - allowing user to create grandchild when creating child

### ISSUE THREAD

**Issue by dekhaus** [sferik/rails_admin#2566]
Hi All

     I realize this is probably a horrible idea from a UI perspective - but I have a client who wants it.

     When creating a Child of a Parent record - RailsAdmin allows said Child to be created in a modal (let's call this modal the 'child creation modal'.  Is there a way to enable 'grandchild' records to be created from within the 'child creation modal' - aka allowing another modal (say the 'grandchild creation modal') to be opened from the 'child creation modal' ?

     Hope that make sense.

Thanks
Dave

**Comment by sax25** [comment]
Yeah - having hit the same wall. After hours of web-reading I'm not closer to a solution.

I _think_ it has something to do with rails_admin not issueing pre-generated ids to all new-forms, therefore the associated children lack the value for the foreign key...

And by the way - I think it's a **great** idea to add multiple layered attributes (children + grandchildren) in one create - run. Makes absolute sense for inputting hierarchical data...
